lib/modules/manager/argocd/readme.md

```diff
@@ -1,34 +1,31 @@
-The `argocd` manager has no `fileMatch` default patterns, so it won't match any files until you configure it with a pattern.
-This is because there is no commonly accepted file/directory naming convention for argocd YAML files and we don't want to check every single `*.yaml` file in repositories just in case any of them have ArgoCD definitions.
+To use the `argocd` manager you must set your own `fileMatch` pattern.
+The `argocd` manager has no default `fileMatch` pattern, because there is no common filename or directory name convention for Argo CD YAML files.
+By setting your own `fileMatch` Renovate avoids having to check each `*.yaml` file in a repository for a Argo CD definition.
 
-If most `.yaml` files in your repository are argocd ones, then you could add this to your config:
+If you need to change the versioning format, read the [versioning](../../../modules/versioning.md) documentation to learn more.
 
-```json
+Some configuration examples:
+
+```json title="Most .yaml files in your repository are for Argo CD"
```

Comment:
I don't know if `"Most .yaml files in your repository are for Argo CD"` is good enough to use the example config.

Say you have GitHub Actions config files in your repository, those _also_ use `.yaml`. So when you set the `fileMatch` for your Argo CD files, I supect Renovate messes up the GitHub Actions `.yaml` files when it updates, or gets confused.

I suspect that only when _all_ `.yaml` files are Argo CD files it's a good idea follow this `fileMatch` config example?

Reply:
I think this is ok, perhaps with an "If most" start